pyconbalkan/contact/models.py

```diff
@@ -0,0 +1,32 @@
+from django.db import models
+from django import forms
+from django.forms import ModelForm
+
+
+class Contact(models.Model):
+    name = models.CharField(max_length=256)
+    company = models.CharField(max_length=100, null=True, blank=True)
+    email = models.EmailField()
+    message = models.TextField()
+
+    def __str__(self):
+        contact_str = '{} | {}'.format(self.name, self.email)
+        if self.company:
+            return '{} | {}'.format(contact_str, self.company)
+        return contact_str
```

Comment:
@nadaj please, create the `__str__` representation for the models. We are using the django admin and it's how they will see the representation of the models 😃

Reply:
@katiayn No problem! 👍